externals/llvm-external-projects/torch-mlir-dialects/include/torch-mlir-dialects/Dialect/Tcp/IR/TcpOps.td

```diff
@@ -158,6 +158,27 @@ def Tcp_DivFOp : Tcp_BinaryElementwiseOp<"divf", [SameOperandsAndResultElementTy
   let assemblyFormat = "$in1 `,` $in2 attr-dict `:` type($in1) `,` type($in2) `->` type($out)";
 }
 
+def Tcp_ConstOp : Tcp_Op<"const", [ConstantLike, Pure]> {
+  let summary = "Constant op";
+
+  let description = [{
+    An op that produces constant tensor (of floating point or signless integer type)
+    that are used in other ops.
```

Comment:
Nit: I don't think the `that are used in other ops` adds anything -- I'd just drop it.